### DISCUSSION TITLE
Using SigNoz? Share your feedback with us

### DISCUSSION THREAD

**Discussion by pranay01** [SigNoz/signoz#335] [General]
Feel free to add any feedback you may have, we are always eager to learn more from users and developers on how we can improve the project

**Comment by akashbabu91** [comment]
I've been using this for debugging in lower environments like dev, staging and it's immensely helpful and prevents the tool fragmentation in terms of telemetry. really appreciate the work you guys are doing and would love to see this go v1.

**Reply by pranay01** [reply]
Thanks for the note @akashbabu91 Really appreciate it. Which features in SigNoz do you use today? Which features do you love?

**Reply by akashbabu91** [reply]
Hey @pranay01, I'm currently using the distributed tracing and metrics using otel, the automated dashboards per service which shows external calls, infra latencies etc is excellent and the service map too. Those are 2 main features I love.

**Reply by pranay01** [reply]
Awesome! Any suggestions on improvements in the product which would be helpful to you? Or any friction points you face today?

**Reply by akashbabu91** [reply]
Absolutely, I am currently running 0.22 of SigNoz which is lacking something like a metrics explorer, where I could just go in and select some of those to create custom dashboards, the explorer on it's own for metrics would be quite useful for me, this is currently possible only via queries directly to the clickhouse db I believe.

**Reply by pranay01** [reply]
If you to Dashboards section from left panel and go to Add panel, you should be able to create custom dashboards. We also have a query builder for building these dashboards and you don't need to use ClickHouse queries.

Does it not satisfy your usecase?

Agree though that this could be more easily accessible

![dashboards-metrics](https://github.com/SigNoz/signoz/assets/504541/142f62f7-2bf9-48a6-aae1-e14e178148be)

**Reply by akashbabu91** [reply]
awesome, thanks for this info @pranay01 , appreciate it 💯

**Comment by Charan2602** [comment]
Hi Team,

I have installed signoz on one vm and also i have instrumented the application to see the traces and services i have given my signoz vm url in the application code. after that i am succesfully viewing my traces in the signoz dahsnboard.

My query is that:
I want to check the cpu, memory metrics of multiple vm's, how can i configure this.

**Reply by pranay01** [reply]
Hey, you need to enable hostmetrics receiver, you can find more detailed docs here https://signoz.io/docs/userguide/hostmetrics/

**Reply by Charan2602** [reply]
Hi @pranay01 Thanks for sharing, i will look into it if any concern i will reach you out.

**Reply by Charan2602** [reply]
Hi @pranay01 i hope you are doing well.
i have followed the documentation that you have shared after importing the json file in the signoz dashbaord , i am getting like the below:
![Screenshot from 2024-02-26 12-07-49](https://github.com/SigNoz/signoz/assets/105963636/1d1ebe27-bf3d-4f94-b08a-90b5ff98deec)

**Comment by Charan2602** [comment]
Hi @pranay01 

I have instrumented the code to emit metrics every 15s once to the following URL.
URL: http://<private_ipaddress>:4318/v1/metrics

and then i created the dashboard and i am able to see the memory usage and messages. but suddenly it is not showing what may be the issue.

**Comment by Manalisatpute** [comment]
I am facing an issue with SigNoz where data is not showing on my custom dashboard after making changes to the otel-collector.yaml file. The services appear to be running fine, but no telemetry data is being reflected. Can you guide me on what configurations might be causing this issue or how to debug it?
